packages/fxa-payments-server/public/locales/en-US/main.ftl

```diff
@@ -44,7 +44,7 @@ product-plan-details-heading = Let's set up your subscription
 # Variables:
     #  $intervalCount (Number) - The interval between payments, in days.
```

Comment:
It looks like these comments still need to be un-indented. You can take a look at the [Fluent Playground](https://projectfluent.org/play/?id=78785dd1cafe0723fb2a9367dcddf201) to see the syntax errors they're causing right now when they're indented.

Reply:
Oh wow! The Fluent Playground is amazing 😍